solidity/contracts/IBancorNetworkPathFinder.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.4.26;
+import './token/interfaces/IERC20Token.sol';
+
+/*
+    Bancor Network Path Finder interface
+*/
+contract IBancorNetworkPathFinder {
```

Comment:
@yudilevi : So why not `IConversionPathFinder`?

Reply:
for the sake of consistency - will rename these together right after the merge